webapp/locales/en.json

```diff
@@ -31,6 +31,9 @@
         "title": "Join Human Connection!",
         "form": {
           "description": "To get started, enter your email address:",
+          "terms-and-condition": "Our <a href=\"/terms-and-conditions\"><ds-text bold color=\"primary\" > terms of conditions!</ds-text></a> apply!",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-15T11:53:41Z" title="Tuesday, October 15th 2019, 1:53:41 pm +02:00">Oct 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -31,6 +31,9 @@
         "title": "Join Human Connection!",
         "form": {
           "description": "To get started, enter your email address:",
+          "terms-and-condition": "Our <a href=\"/terms-and-conditions\"><ds-text bold color=\"primary\" > terms of conditions!</ds-text></a> apply!",
```


I would not use exclamation marks. (!)

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-10-15T12:58:10Z" title="Tuesday, October 15th 2019, 2:58:10 pm +02:00">Oct 15, 2019</time>_

---


That's agreed with martin. 
because you have to put a checkmark at the registration .. there is only a hint with a link to the infosource at this place. 

at this place no little checkers should be set. 

but no worries I wanted to put them also there ran. 

please talk to martin about such suggestions.